examples/standalone-mounted/main.tf

```
@@ -33,7 +33,7 @@ resource "aws_key_pair" "main" {
 module "secrets" {
   source = "../../fixtures/secrets"
   tfe_license = {
-    name = "tfe-license"
+    name = "${var.friendly_name_prefix}-license"
```

If you want this to be unique for reuse purposes, I would suggest:
```suggestion
    name = var.license_secret_name
```

---

I would still want to use "${var.friendly_name_prefix}-license" because then we could easily relate a license to a particular build and also other components of the build like kms key, ssh keypair follow the same convention. 
we wouldn't need to re-use a name, but names should be unique for each build.